client/app/components/ApplicationArea/ErrorMessage.less

```
@@ -0,0 +1,12 @@
+.error-message-container {
+  width: 100%;
+  padding: 0 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+
+  .error-state {
+    max-width: 1200px;
```

```suggestion
    max-width: 1200px;
    width: 100%;

    @media (min-width: 768px) {
      width: 65%;
    }
```

Or anything else at will, the idea is to keep a minimum/tending width to the error div (simulating the `.col-md-8`).